00-Basics/11-ListFactors/listFactors.cpp

```diff
@@ -0,0 +1,28 @@
+#include<iostream>
+#include<conio.h>
+using namespace std;
+
+int main(){
+	int j=0;
+     int tab[j];
```

Comment:
You create a 0 sized array. Later your code can easily casue segfauilt. Also, there is no need for arrays.